packages/amplify_test/lib/test_models/Address.dart

```diff
@@ -34,183 +35,159 @@ class Address {
   String get line1 {
     try {
       return _line1!;
-    } catch (e) {
-      throw AmplifyCodeGenModelException(
-          AmplifyExceptionMessages
-              .codeGenRequiredFieldForceCastExceptionMessage,
-          recoverySuggestion: AmplifyExceptionMessages
-              .codeGenRequiredFieldForceCastRecoverySuggestion,
-          underlyingException: e.toString());
+    } catch(e) {
+      throw new AmplifyCodeGenModelException(
```

Comment:
These changes don't seem correct

Reply:
This is what codegen generated I think, we usually reformat these files.
Codegen code definitely needs some clean up...

Reply:
I just ran the formatter on all these so hopefully that cleans up the diff.